<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Splitter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="NoteWidget" name="nw" native="true">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>5</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Note which divides keyboard on two groups</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>1st group</string>
    </property>
    <widget class="QComboBox" name="channel1Combo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>25</y>
       <width>64</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="channel1Spin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>65</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Shift Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>40</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>2d group</string>
    </property>
    <widget class="QComboBox" name="channel2Combo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>25</y>
       <width>64</width>
       <height>24</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="channel2Spin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>65</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Shift Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>15</y>
      <width>77</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Split on note</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>15</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Use channel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="inputChCombo">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>10</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>MIDI messages will be received on this channel only</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NoteWidget</class>
   <extends>QWidget</extends>
   <header>notewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
